authz/rbac_translator.go

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
AIUI, this file is forked directly from gRPC (as its an internal package). Should we:

a) add a comment that this is the case (i.e., forked from <this location>)
b) check with the gRPC folks whether they intend for this package to be public in the future -- I presume its still experimental which is why it's a private package?

Reply:
Agree with (a).

For (b) wouldn't we contribute this to gRPC eventually? I don't think the rbac code is experimental but it was not intended to be relied upon externally.

Reply:
On (b): you're suggesting that we contribute interzeptors to gRPC? I'm not sure that our use case will be common enough that it would be part of the core libraries -- WDYT? Did you talk with the gRPC folks at all?